src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+const checkStatusReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return state.filter((category) => category.id === action.id);
+
+    default:
+      return state;
+  }
+};
+
+const nextCategoryId = 0;
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+  id: nextCategoryId + 1,
+});
```

Comment:
- [ ] This is not creating a `Check Status` string. Kindly make it something like below ⬇️

```
export const checkStatus = () => ({
  type: CHECK_STATUS,
  payload: 'Check Status',
});
```